src/adapter/threads.ts

```diff
@@ -1265,17 +1266,49 @@ export class Thread implements IVariableStoreDelegate {
    */
   async _handleSourceMapPause(scriptId: string, brokenOn: Cdp.Debugger.Location): Promise<boolean> {
     this._pausedForSourceMapScriptId = scriptId;
-    const timeout = this._sourceContainer.sourceMapTimeouts().scriptPaused;
+    const perScriptTimeout = this._sourceContainer.sourceMapTimeouts().scriptPaused;
+    const timeout =
+      perScriptTimeout + this._sourceContainer.sourceMapTimeouts().extraCumulativeScriptPaused;
```

Comment:
I make this a single `pauseForSourcemap` option that defaults to `1000ms` rather than adding an extra property that adds on top of the default; easier to reason about.

Reply:
Clarified in chat